classes/IssuersPayByBank.php

```diff
@@ -92,4 +92,36 @@ public function getSelectedIssuerLogo()
             return 'PayByBank.gif?v';
         }
     }
+    public function orderByPopularity($issuers = []){
```

Comment:
remove this function, you are creating the list, so create a ordered one, you need function only if the list is from SDK unsorted